addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -108,6 +108,10 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
             };
         }
         this.tours[tour.name] = tour;
+        // Overriding a tour should reset its active tooltip and current step.
+        if (options.saveAs) {
+            this._register(true, tour, tour.name);
+        }
```

Comment:
I am removing the last commit of this PR as I don't understand this bunch of code and testing it does not solve everything. If I am not mistaken: with this, the configurator tour works, without this, it's the theme tours that work :shrug: Without this commit, no tour shows up, but I prefer to fix everything related to configurator/theme tours in one go, especially since this code here is very strange (register is not supposed to call _register, that's the point if I do remember correctly).